adapter/model.go

```diff
@@ -139,186 +158,255 @@ const (
+)
+
+const (
+	WebhookEventType_API_AUTH            WebhookEventType = "API_AUTH"
+	WebhookEventType_API_VERIFY_AND_AUTH WebhookEventType = "API_VERIFY_AND_AUTH"
+	WebhookEventType_CHECKOUTFORM_AUTH   WebhookEventType = "CHECKOUTFORM_AUTH"
+	WebhookEventType_THREEDS_VERIFY      WebhookEventType = "THREEDS_VERIFY"
+)
+
+const (
+	WebhookStatus_SUCCESS WebhookStatus = "SUCCESS"
+	WebhookStatus_FAILURE WebhookStatus = "FAILURE"
+)
+
+const (
+	PosStatus_DELETED     PosStatus = "DELETED"
+	PosStatus_PASSIVE     PosStatus = "PASSIVE"
+	PosStatus_ACTIVE      PosStatus = "ACTIVE"
+	PosStatus_REFUND_ONLY PosStatus = "REFUND_ONLY"
+	PosStatus_AUTOPILOT   PosStatus = "AUTOPILOT"
+)
+
+const (
+	PosIntegrator_YKB               PosIntegrator = "YKB"
+	PosIntegrator_GARANTI           PosIntegrator = "GARANTI"
+	PosIntegrator_ISBANK            PosIntegrator = "ISBANK"
+	PosIntegrator_AKBANK            PosIntegrator = "AKBANK"
+	PosIntegrator_ZIRAATBANK        PosIntegrator = "ZIRAATBANK"
+	PosIntegrator_ZIRAATBANK_INNOVA PosIntegrator = "ZIRAATBANK_INNOVA"
+	PosIntegrator_KUVEYTTURK        PosIntegrator = "KUVEYTTURK"
+	PosIntegrator_HALKBANK          PosIntegrator = "HALKBANK"
+	PosIntegrator_DENIZBANK         PosIntegrator = "DENIZBANK"
+	PosIntegrator_VAKIFBANK         PosIntegrator = "VAKIFBANK"
+	PosIntegrator_FINANSBANK        PosIntegrator = "FINANSBANK"
+	PosIntegrator_FINANSBANK_ASSECO PosIntegrator = "FINANSBANK_ASSECO"
+	PosIntegrator_FIBABANK          PosIntegrator = "FIBABANK"
+	PosIntegrator_FIBABANK_ASSECO   PosIntegrator = "FIBABANK_ASSECO"
+	PosIntegrator_ANADOLUBANK       PosIntegrator = "ANADOLUBANK"
+	PosIntegrator_PARAM_POS         PosIntegrator = "PARAM_POS"
+	PosIntegrator_IYZICO            PosIntegrator = "IYZICO"
+	PosIntegrator_SIPAY             PosIntegrator = "SIPAY"
+	PosIntegrator_PAYNET            PosIntegrator = "PAYNET"
+	PosIntegrator_PAYTR             PosIntegrator = "PAYTR"
+	PosIntegrator_BIRLESIK_ODEME    PosIntegrator = "BIRLESIK_ODEME"
+	PosIntegrator_MOKA              PosIntegrator = "MOKA"
+	PosIntegrator_STRIPE            PosIntegrator = "STRIPE"
+	PosIntegrator_TEB               PosIntegrator = "TEB"
+	PosIntegrator_IPARA             PosIntegrator = "IPARA"
+	PosIntegrator_OZAN              PosIntegrator = "OZAN"
+	PosIntegrator_BRAINTREE         PosIntegrator = "BRAINTREE"
+	PosIntegrator_NKOLAY            PosIntegrator = "NKOLAY"
+	PosIntegrator_FAKE              PosIntegrator = "FAKE"
 )
 
 const (
-	API_AUTH            WebhookEventType = "API_AUTH"
-	API_VERIFY_AND_AUTH WebhookEventType = "API_VERIFY_AND_AUTH"
-	CHECKOUTFORM_AUTH   WebhookEventType = "CHECKOUTFORM_AUTH"
-	THREEDS_VERIFY      WebhookEventType = "THREEDS_VERIFY"
+	PosUserType_API PosUserType = "API"
 )
 
 const (
-	WebhookStatusSUCCESS WebhookStatus = "SUCCESS"
-	WebhookStatusFAILURE WebhookStatus = "FAILURE"
+	PosOperationType_STANDARD PosOperationType = "STANDARD"
```

Comment:
Sadly, this is our bad take for Go not having enums. We prefer having same information available in all of our clients, and opted to include our enums like this rather than documenting it and having go client integration flow work differently.